### ISSUE TITLE
ms_analysis.py tweak to allow for different head3.lst path

### ISSUE THREAD

**Issue by CatChenal** [GunnerLab/Stable-MCCE#260]
Right now, the `read_conformers` function in `bin/ms_analysis.py` has "head3.lst" hard-coded in the `open` call [L316](https://github.com/GunnerLab/Stable-MCCE/blob/2cc4355635382db7c8867f1467c9a1abc4294e8e/bin/ms_analysis.py#L316):

```
def read_conformers():
    conformers = []
    lines = open("head3.lst").readlines()
    lines.pop(0)
    for line in lines:
        conf = Conformer()
        conf.load_from_head3lst(line)
        conformers.append(conf)

    return conformers
```
I propose the following non-breaking fix:

```
def read_conformers(head3_path):
    conformers = []
    lines = open(head3_path).readlines()
    lines.pop(0)
    for line in lines:
        conf = Conformer()
        conf.load_from_head3lst(line)
        conformers.append(conf)

    return conformers

conformers = read_conformers("head3.lst")
```

**Outcome of the change**:
Users of the `ms_analysis.py` module can obtain the conformers list from head3.lst with a different path, yet the former behavior is unchanged. 
Example:
```
import ms_analysis as msa
from pathlib import Path

mcce_dir = Path("some/path/to/MCCE/output/")
head3 = mcce_dir.joinpath("head3.lst")

# redo conformer loading with non-standard path:
conformers = msa.read_conformers(head3)
[...]

```

**Comment by CatChenal** [comment]
Actually, above code is breaking: need a try/except block in ms_analysis.py to catch file not found.

**Comment by CatChenal** [comment]
Addressed by PR #261
